umbral/umbral.py

```diff
@@ -496,17 +471,41 @@ def encrypt(self, pub_key: UmbralPublicKey, data: bytes):
 
         return enc_data, capsule
 
-    def decrypt(self, priv_key: UmbralPrivateKey, capsule: Capsule, enc_data: bytes):
+    def _open_capsule(self,
+                      capsule: Capsule,
+                      opener_private_key: UmbralPrivateKey,
+                      capsule_maker_pub_key: UmbralPublicKey,
+                      pre):
```

Comment:
Opener_private_key is OK, but capsule_maker_pub_key it's not, since in principle anyone (even if they don't have any keys) can be a capsule maker. Anyway, this is just a comment on naming. Let's simply put it in our TODO list.